tools/build-tools/src/bumpVersion/bumpVersionCli.ts

```diff
@@ -265,6 +265,15 @@ async function main() {
             fatal(`Local repo is dirty\n${status}`);
         }
 
+        console.log("Running Policy Check with Resolution(fix)");
+        await exec(`node ${__dirname}\\..\\repoPolicyCheck\\repoPolicyCheck.js -r -q`,resolvedRoot,"policy-check:fix failed");
+
```

Comment:
this is probably too aggressive, probably only want this for branch, release and bump

Reply:
Yep... please scope this to only the commands that needs it.

Reply:
And add comment about why and what this is try to accomplish.